apps/frontend/src/components/cards/controltable/ControlTable.vue

```diff
@@ -129,7 +129,7 @@
           :control="item.control"
           :expanded="expanded.includes(item.key)"
           :show-impact="showImpact"
-          :viewed-controls="viewedControlIds"
+          :viewed-controls="[...viewedControlIds]"
```

Comment:
does this spread operation into an array still need to be done when we're not doing the set thing anymore?

Reply:
I removed it.  It was left over from previous changes.